packages/liferay-npm-scripts/src/config/babel.json

```diff
@@ -4,7 +4,17 @@
 			"plugins": ["transform-react-remove-prop-types"]
 		}
 	},
-	"presets": ["@babel/preset-env", "@babel/preset-react"],
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"targets": {
+					"browsers": ["defaults"]
+				}
+			}
+		],
+		"@babel/preset-react"
+	],
```

Comment:
Could you tell if this makes any noticeable difference in our current DXP output?

Reply:
Looking at just `frontend-js-web`

Current Bundle:
`global.bundle.js  138 KiB`

With differential build:
`global.bundle.js  138 KiB`
`modern_global.bundle.js  129 KiB`

So really only a 9kb difference... 

I also tested with lighthouse and the results didn't seem noticeable. Although it could be because I am only testing the `frontend-js-web` module.

Reply:
Well... that's a bit disappointing, isn't it? 😂

Reply:
I did notice that for some reason the modern build isn't being minified, so that may have part to do with its size. Im looking into why it doesnt compress like the other build.